src/main/java/duke/tasks/Task.java

```diff
@@ -0,0 +1,42 @@
+package main.java.duke.tasks;
+
+/**
+ * Represents a task.
+ */
+public class Task {
+    private String task;
+    private boolean isDone;
+
+    /**
+     * Creates a Task instance that has not been done.
+     *
+     * @param task A string containing task details.
+     */
+    Task(String task) {
+        this.task = task;
+        this.isDone = false;
+    }
+
+    /**
+     * Marks task as done.
+     */
+    public void doTask() {
+        this.isDone = true;
+    }
+
+    @Override
+    public String toString() {
+        String check = isDone ? "✓" : "✗";
+        return String.format("[%s] %s", check, task);
+
+    }
+
+    public String saveToString() {
```

Comment:
I like the way you used %s %c for your string representations, ive never thought of doing this before!! It looks great:)